project.clj

```diff
@@ -45,16 +37,14 @@
                               "-b" "./examples/ring_app/bootstrap.cfg"
                               "-c" "./examples/ring_app/ring-example.conf"]}
                    :source-paths ["examples/ring_app/src"]
-                   :dependencies [[puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty-version]
-                                  [puppetlabs/trapperkeeper ~tk-version :classifier "test" :scope "test"]
-                                  [puppetlabs/kitchensink ~ks-version :classifier "test"]
-                                  [org.clojure/tools.namespace "0.2.10"]]}
+                   :dependencies [[puppetlabs/trapperkeeper-webserver-jetty9]
+                                  [puppetlabs/trapperkeeper nil :classifier "test" :scope "test"]
+                                  [puppetlabs/kitchensink nil :classifier "test" :scope "test"]]}
              :testutils {:source-paths ^:replace ["test"]}}
 
   ;; this plugin is used by jenkins jobs to interrogate the project version
-  :plugins [[lein-project-version "0.1.0"]
-            [lein-release "1.0.5" :exclusions [org.clojure/clojure]]
-            [puppetlabs/i18n "0.4.3"]]
+  :plugins [[lein-parent "0.3.1"]
```

Comment:
I think 0.3.1 is the version I want for `lein-parent`.  That's what we're using across other projects so far.  0.3.3 is the latest for `clj-parent`, which I have in the `parent-project.coords` section above.  Wish those two versions weren't so close together.  I keep confusing the two.  😕